skip-tests.json

```diff
@@ -16,7 +16,12 @@
         }
     },
     "alpaca": {
```

Comment:
@ttodua the pr title is `fix okcoin` but there are a bunch of other fixes here. Is this expected?